README.md

```diff
@@ -16,6 +16,12 @@ git clone --recursive https://github.com/FuelLabs/docs-hub
 cd docs-hub
 ```
 
+If your project's documentation does not reside on the default branch, open the `.gitmodules` file and add/change the `branch = ...` field to point to the corresponding branch. Make sure you update the submodules by using the new branch:
+
+```sh
+git submodule update --recursive --remote
```

Comment:
I don't think you need the `--recursive` flag here. Also, I would note this command will update all of the submodules. To just update a certain one, you can add the path at the end, like `git submodule update --remote docs/fuel-indexer`

Reply:
Got it, I'll make the changes. 👌🏽